<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<comment> Begin CVS Header </comment>
<comment>   $Source: /Volumes/Home/Users/shoops/cvs/copasi_dev/copasi/UI/CQEFMResultWidget.ui,v $ </comment>
<comment>   $Revision: 1.5 $ </comment>
<comment>   $Name:  $ </comment>
<comment>   $Author: shoops $ </comment>
<comment>   $Date: 2011/03/07 19:37:50 $ </comment>
<comment> End CVS Header </comment>

<comment> Copyright (C) 2011 - 2010 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and The University </comment>
<comment> of Manchester. </comment>
<comment> All rights reserved. </comment>

 <class>CQEFMResultWidget</class>
 <widget class="CopasiWidget" name="CQEFMResultWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Elementary Modes Result Window</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="mpLblResult">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:16px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:x-large; font-weight:600;&quot;&gt;Elementary Modes Result&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnSave">
       <property name="text">
        <string>Save to File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="mpLblFluxModes">
       <property name="text">
        <string>Flux Modes</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mpEditFluxModes">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="mpSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QTabWidget" name="mpTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="CQEFMListWidget" name="mpEFMListWidget">
      <attribute name="title">
       <string>List</string>
      </attribute>
     </widget>
     <widget class="QTableView" name="mpNetReactions">
      <attribute name="title">
       <string>Net Reactions</string>
      </attribute>
     </widget>
     <widget class="QTableView" name="mpReactionMatrix">
      <attribute name="title">
       <string>EFM vs Reactions</string>
      </attribute>
     </widget>
     <widget class="QTableView" name="mpSpeciesMatrix">
      <attribute name="title">
       <string>EFM vs Species</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CQEFMListWidget</class>
   <extends>QWidget</extends>
   <header>CQEFMListWidget.h</header>
  </customwidget>
  <customwidget>
   <class>CopasiWidget</class>
   <extends>QWidget</extends>
   <header>copasi/UI/copasiWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mpEditFluxModes</tabstop>
  <tabstop>mpBtnSave</tabstop>
  <tabstop>mpTabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mpBtnSave</sender>
   <signal>clicked()</signal>
   <receiver>CQEFMResultWidget</receiver>
   <slot>slotSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
